### PR TITLE
adding a paragraph or two about the other bucket for a chart

### LINKED ISSUE #59: "Other" bin in Charts
What does the “other” bin in charts, etc., mean? What data goes into that? How is it counted? How do we decide which sets to put into it?

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
closes #59

This just adds a paragraph to the description, the existing examples showcase chart really well.

### CODE REVIEW

File: Search/search.tex [hunk 1]
```diff
@@ -1105,6 +1105,8 @@ \subsection{Chart Renderer}
 | chart count by username limit 10
 \end{Verbatim}
 
+The ``other'' data group is a combined calculation of all remaining data groups that will not fit in the specified set of data series.  In the above examples we have limited the data series to 10 using the \code{limit 10} argument to chart, this means that the chart module will first survey the data as it comes in to pick the best data set to include and then add all other data sets to the ``other'' bucket.  Chart calculates the value of the ``other'' data set using the exact same math as the rendered data set, often leading to the ``other'' data set dominating the output of a chart.
```
Comment (ashnwade) [inline Search/search.tex:1108]: "We have limited the data series to 10 using the \code{limit 10} argument to chart, this means..."
That's not really what limit 10 means. This is a more general explanation of what happens regardless of whether limit 10 was applied or not.

Comment (kris-watts-gravwell) [reply Search/search.tex]: updated the wording, PTAL

**Review by ashnwade — COMMENTED** [review]
Your use of "data groups" vs "data series" vs "data sets" here is somewhat confusing.

**Comment by kris-watts-gravwell** [comment]
updated wording and made sure to be consistent with "data series"

**Review by ashnwade — APPROVED** [review]
Updated language for "other" to be more readable and consistent with the wiki.

**Pull request merged**